src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import core.basesyntax.enums.Color;
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
+    public Random getRandom() {
+        return random;
+    }
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[random.nextInt(Color.values().length)].name();
```

Comment:
It's hard to read such code, it's better to create variable `index = random.nextInt(Color.values().length)`  here

Reply:
In my third commit I did exactly the same var, but after reading COMMON MISTAKES I've removed it. I agree with your point of view and I'll fix it.